api/src/services/lottery.service.ts

```diff
@@ -224,4 +257,325 @@ export class LotteryService {
 
     return ordinalArray;
   }
+
+  /**
+   *
+   * @param queryParams
+   * @param req
+   * @returns generates the lottery export file via helper function and returns the streamable file
+   */
+  async lotteryExport<QueryParams extends ApplicationCsvQueryParams>(
+    req: ExpressRequest,
+    res: Response,
+    queryParams: QueryParams,
+  ): Promise<StreamableFile> {
+    const user = mapTo(User, req['user']);
+    await this.authorizeLotteryExport(user, queryParams.listingId);
+
+    const workbook = new Excel.Workbook();
+
+    const filename = join(
+      process.cwd(),
+      `src/temp/lottery-listing-${queryParams.listingId}-applications-${
+        user.id
+      }-${new Date().getTime()}.xlsx`,
+    );
+
+    const zipFilePath = join(
+      process.cwd(),
+      `src/temp/lottery-listing-${queryParams.listingId}-applications-${
+        user.id
+      }-${new Date().getTime()}.zip`,
+    );
+
+    await this.createLotterySheet(workbook, {
+      ...queryParams,
+      includeDemographics: true,
```

Comment:
Curious if this "includeDemographics" should be passed in from the frontend as part of the queryParams?

Reply:
thats a good question, my read was that the export would always have the demographic fields, but maybe I'm wrong there 

I'll check in with sarah and reply here with what she says!